indico/modules/rb/client/js/common/roomSearch/reducers.js

```diff
@@ -12,7 +12,7 @@ import {requestReducer} from 'indico/utils/redux';
 import {filterReducerFactory} from '../filters';
 import {mapSearchReducerFactory} from '../map';
 import {roomSearchActionsFactory} from './actions';
-import {actions as bookRoomActions} from '../../modules/bookRoom';
+import * as bookRoomActions from '../../modules/bookRoom/actions';
```

Comment:
Even though the previous commit was to avoid direct imports clashing with importing from the index. There isn't really anything I can do about this particular case.

Removing the actions export in bookRoom would result in multiple changes that aren't really effective since `roomSearch` is the only import that creates a cycle. I think this is reasonable.